src/modules/squeeze/mapper/create-squeeze.mapper.ts

```diff
@@ -0,0 +1,22 @@
+import { SqueezeRequestDto } from '../dto/squeeze.dto';
+import { Squeeze } from '../entities/squeeze.entity';
+
+export class CreateSqueezeMapper {
+  static mapToEntity(dto: SqueezeRequestDto): Squeeze {
+    if (!dto) {
+      throw new Error('SqueezeRequestDto is required');
+    }
+
+    const squeeze = new Squeeze();
```

Comment:
this can be achieved with 2 to 3 lines of code with Object.assign

Reply:
yea, makes sense. updated